include/deal.II/grid/reference_cell.h

```diff
@@ -364,6 +364,14 @@ class ReferenceCell
   std_cxx20::ranges::iota_view<unsigned int, unsigned int>
   face_indices() const;
 
+  /**
+   * Return an object that can be thought of as an array containing the indices
+   * of the givent cell's faces of the desired type @p face_ref_type. Indices
+   * form a continuous range that is empty in case no face of such type exists.
+   */
+  std_cxx20::ranges::iota_view<unsigned int, unsigned int>
+  face_indices_by_type(const ReferenceCell &face_ref_type) const;
```

Comment:
Could you show me where you use this function in the bigger patch? We typically only need to loop over face indices, not face types.